root/.stylelintrc

```diff
@@ -0,0 +1,27 @@
+{
+  "plugins": [
+    "stylelint-order"
+  ],
+  "rules": {
+    "block-no-empty": null,
+    "color-no-invalid-hex": true,
+    "comment-no-empty": true,
+    "declaration-block-single-line-max-declarations": 1,
+    "declaration-colon-space-after": "always",
+    "declaration-colon-space-before": "never",
+    "indentation": 2,
+    "max-empty-lines": 1,
+    "no-descending-specificity": true,
+    "no-duplicate-at-import-rules": true,
+    "no-duplicate-selectors": true,
+    "no-empty-source": true,
+    "no-extra-semicolons": true,
+    "rule-empty-line-before": [ "always", {
+      "except": ["first-nested"],
+      "ignore": ["after-comment"]
+    }],
+    "selector-pseudo-class-no-unknown": true,
+    "selector-pseudo-element-no-unknown": true,
+    "order/properties-alphabetical-order": true
+  }
```

Comment:
these rules are defined here: https://stylelint.io/user-guide/rules/
the plugin `stylelint-order` and its corresponding value of `"order/properties-alphabetical-order": true` allow us to auto alphabetize CSS properties

Reply:
Here are the stylelint rules I had on Fashion Positive. Not sure if any of them would be of value to add here. https://github.com/C2CPII/fashion-positive/blob/master/.stylelintrc

Reply:
@jeremyfrank i dont have access to that repo. could you put it in a gist?

Reply:
I'll just post here. It's not much.

```
{
  "rules": {
    "block-no-empty": true,
    "color-hex-case": "lower",
    "color-hex-length": "short",
    "comment-no-empty": true,
    "declaration-block-semicolon-newline-after": "always",
    "indentation": 2,
    "no-extra-semicolons": true,
    "number-leading-zero": "always",
    "number-no-trailing-zeros": true,
  }
}
```